cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/draft/Draft.java

```diff
@@ -41,9 +41,9 @@ private Draft(ETLConfig config, String previousHash, String name, String descrip
 
   // This should be the default constructor until previousHash and revision are needed
   public Draft(ETLConfig config, String name, String description, ArtifactSummary artifact,
-               String id,
-               long createdTimeMillis, long updatedTimeMillis) {
-    this(config, "", name, description, 0, artifact, id, createdTimeMillis, updatedTimeMillis);
+               String id, long createdTimeMillis, long updatedTimeMillis, String parentVersion) {
+    this(config, "", name, description, 0, artifact,
```

Comment:
Should parentVersion be `@Nullable`?

Reply:
It is nullable, in the base Class `DraftStoreRequest`